src/streaming/XHRLoader.js

```diff
@@ -111,7 +111,7 @@ function XHRLoader(cfg) {
                     );
                 } else {
                     if (config.error) {
-                        config.error(request, 'error', xhr.statusText);
+                        config.error(request, 'error', xhr.status, xhr.statusText);
```

Comment:
Do the other loaders (fragment, xlink) would need their error handlers updated too since this is not backwards-compatible?

Reply:
yes, indeed, I have to update :

- SegmentBaseLoader
- WebmSegmentBaseLoader
- XlinkLoader
- FragmentLoader

In some cases, error parameters are not used at all.